pyproject.toml_future

```diff
@@ -22,7 +22,7 @@ classifiers = [
 ]
 requires-python = ">=3.9"
 dependencies = [
-    "pyaro >=0.0.6",
+    "pyaro[optional] >=0.0.6",
```

Comment:
please use 0.0.13, as soon as it is released: https://github.com/metno/pyaro/pull/55